anthony_romm.cfdg

```diff
@@ -0,0 +1,67 @@
+startshape ALIEN
```

Comment:
Do you have any overall thoughts as to whether or not your image is a "good" choice for the language you're working in? Unlike most of the examples (and other submissions) you don't really have a symmetrical, recursive type image but rather a more "real-world" image.  I made a similar choice

![Dan's bad CF image](https://raw.githubusercontent.com/hmc-cs111-fall2015/what-is-a-DSL/b9006c3287ecef70f1b45cd29f8b4c17a04945b2/daniel_obermiller_omg.png)

but I think yours turned out much nicer :)

One thing I noticed, however, was that the parts I drew recursively were much, much easier to draw.